react/ContactsList/styles.styl

```diff
@@ -59,6 +59,9 @@ $reset-list
 .contact-lastname
     font-weight bold
 
+.contact-avatar
+    flex-shrink 0
```

Comment:
It should be directly on the avatar right ? An avatar should never shrink IMHO.

Reply:
cc @y-lohse

Reply:
Not sure. It should have a fixed size, but it already has a fixed width and height. I don't think the component itself should define how it behaves when it's put inside a flex layout. There are other ways to change the size of a component (grid layout, absolute positions, etc), imo it's the responsibility of the component user to handle those cases, like it's been done here.

Reply:
Yes I don't think the Avatar should know how it should behave in all contexts in which it can be rendered.

Reply:
OK good point, thanks !